src/common/util/include/openvino/util/shared_object.hpp

```diff
@@ -21,6 +21,7 @@ namespace util {
  * @param path Full or relative path to the plugin library
  * @return Reference to shared object
  */
+std::shared_ptr<void> load_shared_object(const char* path, const bool& verify_signature);
```

Comment:
I think we need to update description with new param:)

Reply:
@rkazants Thanks! For the moment it's just a POC to verify CI lib signs are compatible with Runtime signs verification